server/tsconfig.json

```diff
@@ -3,8 +3,8 @@
   "compilerOptions": {
     "outDir": "./dist",
     "rootDir": "./src",
-    "strictFunctionTypes": true,
-    "strictNullChecks": true
+    "strictFunctionTypes": false,
```

Comment:
Why this change here and below? @lindner should confirm.

Reply:
Bad merge, this comes from before Paul flipped 'em, will revert.